TicTacToeConsoleApp/Program.cs

```diff
@@ -0,0 +1,174 @@
+﻿namespace TicTacToeConsoleApp
+{
+    public class Program
+    {
+        private static Dictionary<string, string> board;
+        private static string currentPlayer;
+
+        private static void Main(string[] args)
+        {
+            // Инициализация игрового поля и переменных
+            board = new Dictionary<string, string>();
+
+            currentPlayer = "X";
```

Comment:
Use int to store the player's index
0 or 1.
And a dictionary to store the player's display.
<int, stirng> or array string[]
var player = players[0] \\ player = x
var player = players[1] \\ player = o